GitHubToCBuilder.user.js

```diff
@@ -111,6 +112,11 @@
         output: ['# header1', '### header2 some text', '## header3'],
         testingFunc: getHeadersLines,
       },
+      {
+        input: `s # header1\r\n### header2 some text`,
```

Comment:
Please add another one test. Line ` # header1` (prefixed by space symbol) will be represented by GitHub as header, but it looks your code will not recognize this kind line as Header Line.

Reply:
Done.